routes/users.js

```diff
@@ -24,6 +25,19 @@ function dateDiff(dateTimeA, dateTimeB) {
     return result;
 }
 
+function dateDiff(dateTimeA, dateTimeB) {
+    var momentA = moment(dateTimeA,"YYYY-MM-DD");
+    var momentB = moment(dateTimeB,"YYYY-MM-DD");
+    var result = momentB.diff(momentA, 'days');
+    return result;
+}
+
+function monthName(dateTimeA, dateTimeB) {
+    var momentA = moment(dateTimeA,"YYYY-MM-DD");
+    var check = momentA.month();
+    return (moment.monthsShort()[check]);
```

Comment:
extract the month for datetimeB also and then compare the months and add the corresponding days